### PR TITLE
Move libP2P logs to debug level

### PR DESCRIPTION
**Pull request by zhangchiqing**

### CODE REVIEW

File: network/p2p/tracer/gossipSubScoreTracer.go [hunk 1]
```diff
@@ -252,10 +252,10 @@ func (g *GossipSubScoreTracer) logPeerScore(peerID peer.ID) bool {
 	}
 
 	if snapshot.IsWarning() {
-		lg.Warn().Msg(PeerScoreLogMessage)
+		lg.Debug().Msg(PeerScoreLogMessage)
```
Comment (peterargue) [inline network/p2p/tracer/gossipSubScoreTracer.go:255]: from the function name `IsWarning()`, it seems like this should remain a warning. Is it over reporting?

**Review by peterargue — APPROVED** [review]
these all seem OK to make debug. I added one flag I'd like @yhassanzadeh13's take on

File: network/p2p/tracer/gossipSubScoreTracer.go (hunk 1 above)
Comment (zhangchiqing) [reply network/p2p/tracer/gossipSubScoreTracer.go:255]: I do saw lots of warning log here, which I have mentioned to @yhassanzadeh13 
image

Comment (yhassanzadeh13) [reply network/p2p/tracer/gossipSubScoreTracer.go:255]: It should be a `Warn` level, the reason it appears frequently is the IP collocation factor, which we need to fix it.

**Review by yhassanzadeh13 — APPROVED** [review]
Except two comments, the rest LGTM.

**Pull request merged**